bots/task/testmap.py

```diff
@@ -42,6 +43,7 @@
             'fedora-i386',
             'fedora-testing',
             'rhel-7-8',
+            'continuous-atomic',
```

Comment:
This feels a bit odd, as this can only really work on the rhel-7.x branch, not on master. Can we specify the full pattern including @ here?

Reply:
> Can we specify the full pattern including @ here?

Unfortunately we cannot - it would not work with whatever uses it (since things strip the @ part and check if the part before @ is in testmap. Surely it could be edited to work.

> only really work on the rhel-7.x branch

Then we should put it into `rhel-7.x` branch and not into `manual`.

Reply:
OK, let's do that then for clarity. We don't strictly need c-atomic to succeed on rhel-7.x updates, but it can't hurt (and we still can choose to ignore it).

Reply:
done